src/window/winit_window/frame_input_generator.rs

```diff
@@ -28,6 +30,7 @@ pub struct FrameInputGenerator {
     secondary_finger_id: Option<u64>,
     modifiers: Modifiers,
     mouse_pressed: Option<MouseButton>,
+    gesture_input_devices: HashSet<DeviceId>,
```

Comment:
Why is this necessary exactly? 🤔

Reply:
Yeah, I think it's better to either never do that mapping or always do that mapping. Which one do you prefer? Also, the mapping should happen in `FrameInputGenerator` I think, so a `WindowEvent::MouseWheel` event is mapped to a `Event::PinchGesture` if ctrl is pressed.

Reply:
There should at minimum be a way to differentiate ctrl-scroll on devices where that is the event used for a physical pinch input, so it would probably be better to always map ctrl-scroll to pinch. This remapping should not happen in `FrameInputGenerator`, since that would also affect the events that an embedded egui pane receives. (egui widgets should be able to directly handle native input events and are provided modifier information, so they don't need remapping.)